appveyor.yml

```diff
@@ -23,17 +23,23 @@
         matrix:
             - BOND_BUILD: Python
               BOND_VS: "Visual Studio 14 2015"
+              BOND_VS_NUM: 14
+              BOND_ARCH: 32
               BOND_BOOST: 59
             - BOND_BUILD: C++
               BOND_VS: "Visual Studio 14 2015 Win64"
+              BOND_VS_NUM: 14
+              BOND_ARCH: 64
               BOND_BOOST: 59
             - BOND_BUILD: C#
               BOND_OUTPUT: Properties
               BOND_CONFIG: Debug
             - BOND_BUILD: Doc
             - BOND_BUILD: C++
               BOND_VS: "Visual Studio 12 2013"
-              BOND_BOOST: 55
+              BOND_VS_NUM: 12
+              BOND_ARCH: 32
```

Comment:
Can we build 64-bit for our only 2013 flavor? It's probably slower, but it's what the vast majority of our customers run in production.